openslides_backend/action/relations/user_meeting_ids_handler.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict
+
+from openslides_backend.services.datastore.deleted_models_behaviour import (
+    DeletedModelsBehaviour,
+    InstanceAdditionalBehaviour,
+)
+
+from ...models.fields import Field
+from ...shared.patterns import Collection, FullQualifiedField, FullQualifiedId
+from .calculated_field_handler import CalculatedFieldHandler
+from .typing import ListUpdateElement, RelationUpdates
+
+
+class UserMeetingIdsHandler(CalculatedFieldHandler):
+    """
+    CalculatedFieldHandler to fill the user.meeting_ids.
+    """
+
+    def process_field(
+        self, field: Field, field_name: str, instance: Dict[str, Any], action: str
+    ) -> RelationUpdates:
+        if field_name != "group_$_ids":
```

Comment:
Is this needed? When does it get called with other field names?

Reply:
It will be called with `group_$<meeting_id>_ids`, too. I don't want to use these calls.

Reply:
Ah ok, thats fine then